CHANGELOG.md

```diff
@@ -0,0 +1,364 @@
+
+<a name="0.7.1"></a>
+## [0.7.1](https://github.com/redhat-developer/app-services-operator/compare/0.7.0...0.7.1) (2021-04-21)
+
+### Bug Fixes
+
+* add logging capability to tell us what env is used by operator
+* add explicit env variable pointing to the environment
+
+<a name="0.7.0"></a>
+## [0.7.0](https://github.com/redhat-developer/app-services-operator/compare/0.6.9...0.7.0) (2021-04-21)
+
+### Bug Fixes
+
+* update default url to production ([#178](https://github.com/redhat-developer/app-services-operator/issues/178))
+
+
+<a name="0.6.9"></a>
+## [0.6.9](https://github.com/redhat-developer/app-services-operator/compare/0.6.8...0.6.9) (2021-04-21)
+
+### Bug Fixes
+
+* update case for KafkaConnect unit tests ([#187](https://github.com/redhat-developer/app-services-operator/issues/187))
+* url for the kafka in the UI
```

Comment:
Some commits are not related to the PR. I have disable way to have PR with multiple commits merged so we can keep things clean @secondsun